### PR TITLE
edit linux docs

### PR DESCRIPTION
**Pull request by nomicode**
the complimentary pr for crate/guide/pull/74

### CODE REVIEW

File: docs/install-run/special/linux.txt [hunk 1]
```diff
@@ -4,15 +4,22 @@
...
 
-We recommend that you use our one-step installer that will configure the
-appropriate package repository and install CrateDB for you.
+.. _Debian GNU/Linux: https://crate.io/docs/crate/guide/en/latest/deployment/linux/debian.html
+.. _Ubuntu: https://crate.io/docs/crate/guide/en/latest/deployment/linux/ubuntu.html
+.. _RedHat Linux: https://crate.io/docs/crate/guide/en/latest/deployment/linux/red-hat.html
+
+You can take `the manual approach`_, but we recommend that you use our one-step
+installer that will configure the appropriate package repository and install
+CrateDB for you.
```
Comment (chaudum) [inline docs/install-run/special/linux.txt]: this installer is only recommended for "try out" purposes, not for production deployments.

**Review by chaudum — APPROVED** [review]
otherwise ack

File: docs/install-run/special/linux.txt (hunk 1 above)
Comment (nomicode) [reply docs/install-run/special/linux.txt]: @chaudum made some bigger changes based on your feedback. can you review again? please and thank you!

**Review by chaudum — APPROVED** [review]
lgtm

**Pull request merged**